app/src/main/res/layout/subscription_item.xml

```diff
@@ -1,55 +1,97 @@
-        app:primaryTextColor="?attr/colorOnSecondary"
-        app:primaryTextSize="12sp" />
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <androidx.cardview.widget.CardView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:clickable="false"
+                app:cardCornerRadius="8dp"
+                app:cardElevation="0dp">
+
+                <RelativeLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content">
+
+                    <de.danoeh.antennapod.ui.common.SquareImageView
+                        android:id="@+id/imgvCover"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:scaleType="fitCenter"
+                        android:outlineProvider="background"
+                        squareImageView:direction="width"
+                        tools:src="@tools:sample/avatars" />
+
+                    <TextView
+                        android:id="@+id/countViewPill"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="3"
+                        android:layout_alignParentEnd="true"
+                        android:textSize="12sp"
+                        style="@style/TextPill"
+                        app:primaryText="Test"
+                        app:primaryTextColor="?attr/colorOnSecondary"
+                        app:primaryTextSize="12sp" />
+
+                    <TextView
+                        android:id="@+id/fallbackTitle"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_centerInParent="true"
+                        android:layout_centerHorizontal="true"
+                        android:ellipsize="end"
+                        android:gravity="center"
+                        android:padding="6dp"
+                        android:textColor="?android:attr/textColorPrimary"
+                        tools:text="@string/app_name" />
+
+                </RelativeLayout>
+
+            </androidx.cardview.widget.CardView>
+
+            <TextView
+                android:id="@+id/txtvTitle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:gravity="start"
+                android:text="final text"
+                android:textColor="?android:attr/textColorPrimary"
+                android:lines="2"
+                android:layout_marginTop="4dp"
+                android:paddingHorizontal="4dp" />
```

Comment:
I think that with big cards (2-3/row) the padding looks too small. Material guidelines say a padding of 16dp should be used. Now, that's a bit much. But could we set 8dp if we have 2-3 cards per line?

Similarly, with big cards the text looks oddly small. Taking into consideration also that the title display was requested by visually impaired, I think that it would be great to increase the font size if possible. E.g.:
* Slightly increased font size with 3 cards/row.
* Subheading style with 2 cards/row

![Imagepipe_0.jpg](https://user-images.githubusercontent.com/11229646/213646260-24425fc9-7d2f-4c1b-bb63-b4b31d9a5dd8.jpg)
![unnamed(2).png](https://user-images.githubusercontent.com/11229646/213646816-70addfaa-25a7-4313-a156-ea4974ea99a5.png)

Reply:
Ok will change as below then:
- keep horizontal padding to 8dp when card is 2 to 3 per line, and will keep as is for more than that
- increase font title size if 3 cards/row (size less than subheading type)
- set text style to subheading type if 2 cards/row